llvm/lib/Transforms/Utils/InjectTLIMappings.cpp

```diff
@@ -37,20 +37,39 @@ STATISTIC(NumCompUsedAdded,
 /// vectorizes the CallInst CI with a vectorization factor of VF
 /// lanes. The TLI assumes that all parameters and the return type of
 /// CI (other than void) need to be widened to a VectorType of VF
-/// lanes.
+/// lanes unless the VFABI says otherwise.
```

Comment:
Could you rephrase this sentence ?
I think like something like:
"For each mapping TLI contains a VABI prefix, which contains all information required to create vector function declaration."

Reply:
Done